pkg/resource/aws/testdata/acc/aws_vpc/terraform.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  region = "us-east-1"
+  region = "us-east-2"
```

Comment:
This should never work and should be reverted IMHO @moadibfr @wbeuil. We are targetting the symptoms but not the root cause. And BTW this should be changed in the test case too 

![image](https://user-images.githubusercontent.com/6154987/118945435-bd0c7100-b955-11eb-8e83-71b4d1aaf741.png)

Reply:
Yeah we wanted to try having the test pass this does not mean we consider this a fix for the other test that fail. But this if we isolate this one it would be easier to find the one that exhaust the vpc limit. Also it's not armfull to test on different region.

I didn't remember the aws_region but it's strange that the test still pass thought ? I tried it in my env and it runs ok ?1